assets/src/stories-editor/components/index.js

```diff
@@ -10,7 +10,9 @@ export { default as StoryControls } from './story-controls';
 export { default as Shortcuts } from './shortcuts';
 export { default as StoryBlockDropZone } from './story-block-drop-zone';
 export { default as StoryBlockMover } from './block-mover';
-export { default as TemplateInserter } from './template-inserter';
+// @see https://github.com/ampproject/amp-wp/issues/3211
+// export { default as TemplateInserter } from './template-inserter';
```

Comment:
FWIW I commented this line of code so that the component would not accidentally end up in the final bundle.